app/views/registers/index.html.haml

```diff
@@ -1,85 +1,125 @@
     %li.breadcrumbs__item.breadcrumbs__item--active
-      = link_to 'Get data', '#content', 'aria-current' => 'page'
+      = link_to 'Register collection', '#content', 'aria-current' => 'page'
 
-%main#content.container{role:'main'}
-  .grid-row
-    .column-two-thirds
-      %h1.heading-large
-        Registers available
-      %p.heading-large
-        %span.heading-secondary Registers are up-to-date lists of information that are managed and approved by government. You can use this data to build and maintain your service.
-      %p You can also see #{link_to 'registers that are coming soon', registers_in_progress_path}.
-    .column-one-third
-      = form_tag registers_path(anchor: 'content'), class: 'records-search', method: 'get' do
-        = label_tag :q, 'Search', class: 'visually-hidden'
-        = search_field_tag :q, @search_term, class: 'search-input', placeholder: 'Search', autocomplete: 'off'
-        = submit_tag 'Search', name: nil, class: 'search-submit'
+.govuk-width-container
+  %main{class: 'govuk-main-wrapper govuk-!-margin-top-2', role:'main'}
+    .govuk-grid-row
+      .govuk-grid-column-two-thirds
+        %h1{class: 'govuk-heading-l govuk-!-margin-bottom-1'} Government registers collection
+        - unless @search_term.present?
+          %p.govuk-body-l Find registers that are managed and approved by government to use in your service.
+      .govuk-grid-column-one-third
+        = form_tag registers_path(anchor: 'content'), class: 'records-search', method: 'get' do
+          = label_tag :q, 'Search', class: 'visually-hidden'
+          = search_field_tag :q, @search_term, class: 'search-input', placeholder: 'Search', autocomplete: 'off'
+          = submit_tag 'Search', name: nil, class: 'search-submit'
 
-    .column-full
-      - if @registers.present?
-        - if @search_term
-          .search-results
-            %p
-              Results found for <strong>"#{@search_term}"</strong>
-              = link_to 'Reset', registers_path, class: 'reset-link'
-        .grid-row
-          .column-full
-            - if @registers.featured.present? && !@search_term
-              %h2.heading-small Featured
-              %ol.app-c-highlight-boxes
-                - @registers.featured.by_popularity.each do |register|
-                  %li.app-c-highlight-boxes__item-wrapper
-                    .app-c-highlight-boxes__item
-                      %h3.app-c-highlight-boxes__title= link_to register.register_name, register_path(register.slug)
-                      %p.app-c-highlight-boxes__description= register.register_description
-                      - if register.authority.present?
-                        %div{class: "govuk-organisation-logo #{register&.authority&.name.parameterize}"}
-                          %div{class: "logo-container #{crest_class_name(register&.authority&.name.parameterize)}"}
-                            %span= register&.authority&.name
+    .govuk-grid-row#show-by
```

Comment:
Is any of this common to the category pages? If so maybe we can extract a partial here to avoid duplication.

Reply:
I've put a list partial on both the categories and the organisations view on this template.